docs/dev/release_process.rst

```diff
@@ -68,13 +93,17 @@ Set the version in the release branch::
 
 Set the version number in the develop branch *if necessary*.
 
-Push your changes to Github.
-
+Create a pull request on Github to get sign off for the release.
 
-Squash migrations
-~~~~~~~~~~~~~~~~~
+Checklist for sign off:
 
-(explain here)
+- [ ] Tested Debian Installer
+- [ ] Tested Windows Installer
+- [ ] Tested PEX File
+- [ ] Translation files have been updated
+- [ ] Migrations have been squashed where possible
+- [ ] Changelog has been updated
+- [ ] LE Dependencies properly updated
 
```

Comment:
I'm guessing the checklist above is not supposed to be an ordered list? Since testing the installers should be last entries...

Reply:
No, and the assumption is that this PR would have been opened after most of those things have been modified. Will change order to reduce ambiguity.